docs/docs/contribute/docs/01.installations.md

```diff
@@ -0,0 +1,106 @@
+
+![Choose default Git Editor](assets//install-git_default-editor.png)
+
+Retain the other installation defaults.
+
+
+## 1.2 Python
+
+Python is a pre-requisite for both MkDocs and MkDocs Material.
+
+*Python comes pre-installed on most Mac & Linux machines. If it's not on your machine you can get the relevant Python package <a href="https://www.python.org/downloads/" target="_blank">here</a>.*
+
+For windows you can grab the latest Python release installer <a href="https://www.python.org/downloads/windows/" target="_blank">here</a>.
+
+
+Download the installer and follow the on-screen instructions after starting installation. 
+
+Remember to add Python to Path during the installation.
+
+![Add Python to PATH](assets//install-py_add-to-path.png)
+
+And choose to disable the Path Length Limit (just in case your default installation path is in a deeply nested library).
+
+![Disable path length limit](assets//install-py_disable-pll.png)
+
+
+Retain the other default installations settings.
+
+To confirm Python was installed successfully type either Python or Py in the windows command prompt:
+
+> Python
+
+If Python is Installed it will show the version Details 
+
+![Python was installed successfully](assets//install-py_installed.png)
+
+
+
+Typing the following command in the windows command prompt to see where Python is installed:
+
+> where py
+
+![Show where Python is installed](assets//install-py_where-py.png)
+
+
+Also confirm that PIP (Python's installation package) is installed successfully by typing the following command in the windows command prompt (it's installed automatically with Python):
+
+> pip
+
+![Show where Python is installed](assets//install-py_pip-installed.png) 
+
+Official installation instructions for Python on Windows can be found <a href="https://docs.python.org/3/using/windows.html#installation-steps" target="_blank">here</a>.
+
+
+
+## 1.3 MkDocs
+
+MkDocs is a documentation static site generator that makes it easy to create and generate documentation.
+
+Once Python is installed you can install MkDocs using the PIP command from the command prompt:
+> pip install mkdocs
+
+![Install MkDocs](assets//install-mk_install.png) 
+
+You can check the version of MkDocs installation by using the following command:
+> mkdocs --version
+
+![Check MkDocs installation](assets//install-mk_installed.png)
+
+For more details refer to the official installation instructions <a href="https://www.mkdocs.org/user-guide/installation/#installing-mkdocs" target="_blank">here</a> .
+
+
+## 1.4 MkDocs Material
+
+MkDocs Material is a theme for MkDocs and can be installed via the PIP command :
+
+> pip install mkdocs-material
+
+![Install MkDocs-Material](assets//install-mkmat_install.png) 
+
+Refer to the official MkDocs <a href="https://squidfunk.github.io/mkdocs-material/getting-started/" target="_blank">getting-started documentation</a> for more details.
+
+
+
+
```

Comment:
Too many empty lines. Let's try to stay consistent:
- one empty line after titles and headings
- two empty lines before the "includes/license.md" snippet insert.

Reply:
Agreed. Corrected all pages as per this convention.